Mathlib/Data/Option/Basic.lean

```diff
@@ -110,7 +112,7 @@ theorem joinM_eq_join : joinM = @join α :=
   funext fun _ ↦ rfl
 #align option.join_eq_join Option.joinM_eq_join
 
-theorem bind_eq_bind {α β : Type _} {f : α → Option β} {x : Option α} : x >>= f = x.bind f :=
+theorem bind_eq_bind {α β : Type u} {f : α → Option β} {x : Option α} : x >>= f = x.bind f :=
```

Comment:
Another reasonable option for these rare declarations is
```suggestion
theorem bind_eq_bind.{u} {α β : Type u} {f : α → Option β} {x : Option α} : x >>= f = x.bind f :=
```
or
```suggestion
universe u in
theorem bind_eq_bind {α β : Type u} {f : α → Option β} {x : Option α} : x >>= f = x.bind f :=
```
to keep the universe local

Reply:
Thanks for the suggestions!